README.md

```diff
@@ -163,6 +163,37 @@ does not work it may be because you don't have the correct permissions, and
 need to use `sudo make deploy`.
 See the README.md file in the ports/stm32/ directory for further details.
 
+Building under Windows
+----------------------
+
+MicroPython is built using gnu make and requires various gnu utilities ported
+to Windows. MSYS2 should be installed to provide these gnu utilities.
+
+To use frozen bytecode (.mpy modules) you must build mpy-cross for your host OS.
+This requires mingw32 gcc and it dependancies.
+
+Install MSYS2 64 bit from msys2.org
+Run the installer msys2-x86_64-xxxxxxxx.exe
+Follow the instructions on the msys2.org home page to update the package
+database and core modules.
+
+Install the compiler and make:
+
+    pacman -S mingw-w64-i686-gcc 
+    pacman -S make
+
+This will install gcc.exe which is identical to i686-w64-mingw32-gcc
+If you already have MSYS2 / Mingw installed run "gcc -dumpmachine"
+from a command prompt to ensure you have the correct gcc installed.
+Response should be "i686-w64-mingw32"
+
+MicroPython can then be built by running make from a Windows command prompt
```

Comment:
You mean cmd.exe right? How exactly does this work? Does the MSYS2 installer automatcially add the directory containing i686-w64-mingw32-gcc.exe and the likes to the PATH? If so, how would you build a 64bit version where you need the x86_64-w64-mingw32-gcc.exe instead?

Reply:
Yes cmd.exe. I run make from cmd.exe and from an Eclipse makefile project. The path needs to be set. I will do a fresh install and verify if the installer sets the path, and document if necessary. You can still specify an alternative compiler with CROSS_COMPILE=.